engine/src/main/java/com/redhat/cloud/notifications/processors/drawer/DrawerProcessor.java

```diff
@@ -82,7 +81,8 @@ public void process(Event event, List<Endpoint> endpoints) {
         }
 
         // build event thought qute template
-        String renderedData = buildNotificationMessage(event);
+        BundleApplicationEventTypeDTO baet = eventTypeRepository.getEventTypeBaet(event.getEventType().getId());
```

Comment:
The `event` object should already contain the names of the event type, app and bundle. Do we really need that extra DB fetch?